src/helpers/ipfs-daemon.js

```diff
@@ -76,4 +77,27 @@ const setIPFSCORS = (ipfsRpc) => {
   )
 }
 
-module.exports = { startIPFSDaemon, isIPFSCORS, setIPFSCORS }
+const isIPFSRunning = async (ipfsRpc) => {
+  const portTaken = await isPortTaken(ipfsRpc.port)
+
+  if (portTaken) {
+    if (!ipfsNode) ipfsNode = ipfsAPI(ipfsRpc)
+
+    const getId = promisify(ipfsNode.id)
```

Comment:
`ipfsAPI` already returns a promise if a callback is not provided

Reply:
thanks